backend/api/prisonApi.js

```diff
@@ -248,6 +248,9 @@ const prisonApiFactory = client => {
   const getCaseNoteSummaryByTypes = (context, params) =>
     get(context, `/api/case-notes/summary?${mapToQueryString(params)}`)
 
+  const getCaseNote = (context, offenderId, caseNoteId) =>
```

Comment:
Is there a equivalent in the caseNoteApi.js?

Reply:
seems there is  @GetMapping("/{offenderIdentifier}/{caseNoteIdentifier}") shall I use that instead?